src/routes/member.ts

```diff
@@ -24,8 +24,14 @@ router.get(
 		const { student_id, email } = req.query;
 
 		try {
-			const result = await MemberService.isMember(student_id, email);
-			return res.status(200).json({ result });
+			const membership = await MemberService.isMember(student_id, email);
```

Comment:
Hey Ben, to keep the routes simple, could you abstract all logic to the Member Service? For example, rather than have conditionals and logic here, you could simply return a result directly from Member Service. See the other routes for examples.